frappe/public/less/chat.less

```diff
@@ -236,7 +236,7 @@
 
 // ChatBubble
 @frappe-chat-bubble-padding:					@frappe-chat-base-spacing @frappe-chat-base-spacing * 2;
-@frappe-chat-bubble-min-width:                  20%;
+@frappe-chat-bubble-min-width:                  25%;
```

Comment:
this file is already called chat.less, no need to namespace variables!

Reply:
ok.